xdsl/dialects/riscv_func.py

```diff
@@ -58,18 +58,18 @@ class CallOp(IRDLOperation):
     name = "riscv_func.call"
     args: Annotated[VarOperand, riscv.RegisterType]
     func_name: OpAttr[StringAttr]
-    result: Annotated[OptOpResult, riscv.RegisterType]
+    ress: Annotated[VarOpResult, riscv.RegisterType]
```

Comment:
nit: should this be `results` or `res`? `ress` looks a bit weird?

Reply:
results is reserved, res to me implies singular, and ress is weird, alternative suggestions welcome